PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/tree/TreeManager.java

```diff
@@ -232,14 +235,12 @@ private void initializeGraphSizeListeners() {
    * @param root the root of the part of the tree which should be shown.
    */
   @TestId(id = "setRoot")
-  private void setRoot(IPhylogeneticTreeNode root) {
+  public void setRoot(IPhylogeneticTreeNode root) {
```

Comment:
The actual problem with this line is that it is public and has a TestId though. I made it public because there was an exception when calling the method from a test and if you use a testid to call the method the real exception is not shown (a downside to reflection I guess), but I forgot to make it private again.